libpod/container_internal_linux.go

```diff
@@ -336,8 +336,9 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 	}
 
 	g.SetRootPath(c.state.Mountpoint)
-	g.AddAnnotation(crioAnnotations.Created, c.config.CreatedTime.Format(time.RFC3339Nano))
+	g.AddAnnotation(annotations.Created, c.config.CreatedTime.Format(time.RFC3339Nano))
 	g.AddAnnotation("org.opencontainers.image.stopSignal", fmt.Sprintf("%d", c.config.StopSignal))
+	g.AddAnnotation(annotations.ContainerManager, annotations.ContainerManagerPodman)
```

Comment:
I think this would be better to be not set at all or set as libpod or unkonwn, then have podman create/run call into libpod to set the annotations.  That way others vendoring libpod would not indicate they are running podman containers.

Reply:
Yes, I changed it to `annotations.ContainerManagerLibpod` for now to indicate the correct library.